src/signing-panacea-client.ts

```diff
@@ -17,6 +10,8 @@ import { MsgCreateDID, MsgDeactivateDID, MsgUpdateDID } from "./proto/panacea/di
 import { MsgIssueToken } from "./proto/panacea/token/v2/tx";
 import { DIDDocument } from "./proto/panacea/did/v2/did";
 import { IntProto } from "./proto/cosmos/base/v1beta1/coin";
+import { MsgCreateDeal, MsgSellData, MsgDeactivateDeal, DataValidationCertificate } from "./proto/panacea/market/v2/tx";
+import { Coin } from "./proto/cosmos/base/v1beta1/coin";
 
 //////////////////////////////////////////////////////////////////////////////////////////////////
 // TODO: This FeeTable concept is removed on the latest CosmJS (not released yet).
```

Comment:
The FeeTable concept was removed at CosmJS v0.26.0: https://github.com/cosmos/cosmjs/blob/main/CHANGELOG.md#removed-1. That's why I put this comment here.

Please remove all FeeTable-related stuff below such as `panaceaDefaultGasLimits`, and so on.

Or, you can only focus on implementing `x/market`-related functions in this PR. Then, you can do the cosmjs upgrade at next PRs.

Reply:
I agree. PR is too big.
First of all, it would be better to develop with the current version of cosmjs.

Reply:
I think so too. This PR is too big to refactor or review it. 😭 I'll change a few code based on the reviews and re-commit soon.